docs/application_outages.md

```diff
@@ -0,0 +1,17 @@
+### Application outages
+Scenario to block the traffic ( Ingress/Egress ) of an application matching the labels for the specified duration of time to understand the behavior of the service/other services which depend it during the downtime. This helps with the planning the requirements accordingly be it improving the timeouts or tweaking the alerts etc.
+
+##### Sample scenario config
+```
```

Comment:
Would be nice to have a sample here with filled in parameters. The pod_selector variable is a little confusing and in a different format than normal so unsure what to put.

Reply:
Right, updated the docs and sample configuration. Thanks.